mist-api/src/routes/aseqs/post.js

```diff
@@ -11,8 +11,15 @@ module.exports = function(app, middlewares, routeMiddlewares) {
 
 	return [
 		bodyParser(),
-		function(req, res, next) {
+		(req, res, next) => {
 			const aseqIds = req.body
+			aseqIds.forEach((aseqId, i) => {
+				if (!Aseq.validate(aseqId)) {
+					const invalidAseqError = new errors.BadRequestError()
```

Comment:
Set the message when constructing the error:

`new errors.BadRequestError('Invalid item in position ...')`. Also please use ES6 interpolation:

`Invalid item in position ${i}: ${aseqId}`